__tests__/app.test.js

```diff
@@ -159,3 +159,35 @@ describe("POST /api/articles/:article_id/commments", () => {
             });
     });
 });
+
+
+describe("PATCH /api/articles/:article_id", () => {
+    test("202 returns updated article", () => {
+        const patchId = 1;
+        const update = {
+            inc_votes: 1000
+        };
+        return request(app)
+        .patch(`/api/articles/${patchId}`)
+        .send(update)
+        .expect(202)
+        .then(({ body }) => {
+            const { article } = body;
+            expect(article.votes).toEqual(1100);
+        })
+    })
+    test("202 returns updated article", () => {
+        const patchId = 1;
+        const update = {
+            inc_votes: -1000
+        };
+        return request(app)
+        .patch(`/api/articles/${patchId}`)
+        .send(update)
+        .expect(202)
+        .then(({ body }) => {
+            const { article } = body;
+            expect(article.votes).toEqual(-900);
+        })
+    })
+});
```

Comment:
We will need some sad path tests too please:

- 400: invalid article_id e.g. /api/articles/notanumber
- 404: article_id does not exist e.g. /api/articles/9999
- 400: incomplete body e.g. inc_votes value is not a number/undefined